20.9.0 rns (#280)
demisto/content-docs: content-repo/extra-docs/releases/20.9.0.md

# Cortex XSOAR Content Release Notes for version 20.9.0 (98126)
##### Published on 1 September 2020
### New: EmailCommunication Pack v1.0.0
#### IncidentFields
- **Add CC To Email**
#### IncidentTypes
- **Email Communication**
#### Scripts
##### PreprocessEmail  
Preprocessing script for email communication layout.
This script checks if the incoming email contains an Incident ID to link the mail to an existing incident, and tags the email as "email-thread".

For more information about the preprocessing rules, refer to: https://demisto.developers.paloaltonetworks.com/docs/incidents/incident-pre-processing
##### SendEmailReply  
The script sends email massages with the configured mail sender integration.

---

### New: Genians Pack v1.0.0 (Partner Supported)
#### Integrations
##### Genians  
Use the Genians integration to block IP using assign tag and unassign tag.

---

### New: McAfee ESM Pack v1.0.0
#### Integrations
##### McAfee ESM v2  

> Run queries and receive alarms from Intel Security ESM. Support version 10 and above.

Reviewer: McAfee is no long part of Intel.
```suggestion
Run queries and receive alarms from McAfee ESM. Support version 10 and above.
```

---

### New: Microsoft Advanced Threat Analytics Pack v1.0.0
#### Classifiers
#####  Microsoft Advanced Threat Analytics - Classification  
Classifies Microsoft Advanced Threat Analytics suspicious activities
##### Microsoft Advanced Threat Analytics  

#####  Microsoft Advanced Threat Analytics - Incoming Mapper  
Maps Microsoft Advanced Threat Analytics suspicious activity fields.
#### IncidentFields
- **Suspicious Activity End Time**
- **Suspicious Activity ID**
- **Suspicious Activity Severity**
- **Suspicious Activity Start Time**
- **Suspicious Activity Status**
#### IncidentTypes
- **Microsoft ATA Suspicious Activity**
#### Integrations
##### Microsoft Advanced Threat Analytics  
Use Microsoft Advanced Threat Analytics integration to manage suspicious activities, monitoring alerts and entities.

---

> ## New: NTT Cyber Threat Sensor Pack v1.0.0 (Partner Supported)

Reviewer: Problem with table of contents
```suggestion
### New: NTT Cyber Threat Sensor Pack v1.0.0 (Partner Supported)
```

#### Classifiers
##### NTT Cyber Threat Sensor - Classifier  
Classifies NTT Cyber Threat Sensor incidents
##### NTT Cyber Threat Sensor  

##### NTT Cyber Threat Sensor - Incoming Mapper  
Maps incoming NTT Cyber Threat Sensor fields.
#### IncidentFields
- **FAERE Description**
- **Graph Plot**
#### IncidentTypes
- **TD Incident**
#### Integrations
##### NTT Cyber Threat Sensor  
Retrieve alerts and recommendations from NTT CTS
#### Playbooks
##### Handle TD events  
Playbook to enrich TD events

---

### New: PiHole Pack v1.0.0 (Community Supported)
#### Integrations
##### PiHole  
Pi-hole is a network-level advertisement and Internet tracker blocking application which acts as a DNS sinkhole and optionally a DHCP server, intended for use on a private network.

---

### New: QueryAI Pack v1.0.0 (Partner Supported)
#### Integrations
##### Query.AI  
Query.AI is a decentralized data access and analysis technology that simplifies security investigations across disparate platforms without data duplication.

---

### Access Investigation Pack v1.2.2
#### Layouts

> - **layout-edit-Access.json**

Reviewer: @jochman @Itay4 What changed here?

- Adjusted the error handling to use the new API format.

---

### Cisco AMP Pack v1.0.1
#### Integrations
##### Cisco AMP  
Fixed an issue where the `amp_move_computer` command would fail.

---

### Cisco Umbrella Investigate Pack v1.0.1
#### Integrations
##### Cisco Umbrella Investigate  
Fixed an issue where the ***umbrella-ip-dns-history*** command failed when no ip results were found.

---

### Code42 Pack v2.0.3 (Partner Supported)
#### Integrations
##### Code42  
Fix bug where File Category would not map correctly when creating incidents from Code42 alerts.

---

### Cofense Triage Pack v1.1.4 (Partner Supported)
#### Integrations
##### Cofense Triage v2  
Fixed an issue in *cofense-get-attachment* command.

---

### Common Playbooks Pack v1.7.1
#### Playbooks
##### Extract Indicators From File - Generic v2  
Added support for UTF-8 Unicode text files.

---

### Common Scripts Pack v1.2.34
#### Scripts
##### WhereFieldEquals  
Fixed an issue where `WhereFieldEquals` would return a string instead of a list.

##### FeedRelatedIndicatorsWidget  
Fixed an issue where the indicator link value was incorrect.

---

### Common Types Pack v1.8.8
#### Layouts

> - **File Indicator**

Reviewer: what changed?

- **layout-edit-Vulnerability.json**
- **layout-details-Vulnerability.json**

---

### CrowdStrike Falcon Streaming Pack v1.0.7
#### Integrations
##### CrowdStrike Falcon Streaming v2  
- Improved error handling of unsupported media types.
- Improved handling when the stream response client is not completed.
- Maintenance and stability enhancements.

---

### CyberArk Pack v1.0.2
#### Integrations
##### CyberArk PAS  
- Added documentation for the integration.
- Updated the the **cyberark-pas-credentials-verify** command to use the new API.
- Added the ***cyberark-pas-account-get-details*** command.

---

### DUO Admin Pack v2.0.1
#### Integrations
##### DUO Admin  
Updated the integration docker image to the latest version.

---

> ### Email Communication Pack v1.2.0

Reviewer: We have this at the top as a `New` pack. Please unite.

#### Layouts
#### Scripts
##### PreprocessEmail  
You can now view mail attachments for Email Communication incidents.

##### SendEmailReply  
Fixed an issue where the recipients list was parsed incorrectly.

---

### Endace Pack v1.1.0 (Partner Supported)
#### Integrations
##### Endace  
- Added Support for directionless Ip and Port Search. 
- Improved Error handling messages.
- Parameter hostname is now mandatory.

#### Playbooks
##### Endace Search Archive and Download  
This playbook is deprecated. Use Endace Search Archive Download PCAP instead.

##### Deprecated: Endace Search Archive Download PCAP  
This playbook is deprecated. Use Endace Search Archive Download PCAP v2 instead.

##### Endace Search Archive Download PCAP v2  
Added support for directionless ip and port search, user friendly timeframe values, updated input and output variables to the playbook and their definitions.

---

### Expanse Pack v1.1.1 (Partner Supported)
#### Integrations
##### Expanse  
Updated the version number in the user-agent header

---

### GDPR Pack v1.0.5
#### Layouts

> - **layout-edit-GDPR_Data_Breach.json**

Reviewer: What changed here?

- **layout-details-GDPR_Data_Breach.json**

---

### GitHub Pack v1.1.1
#### Integrations
##### GitHub  
Improved implementation of parameters errors handling.

---

### Gmail Pack v1.0.5
#### Integrations
##### Gmail  
Breaking Change: Removed the deprecated *event* argument from the ***gmail-list-users*** command.

---

### IBM QRadar Pack v1.0.8
#### Integrations
##### IBM QRadar  
Added a parameter that enables you to specify the number of addresses to enrich per API call.

---

### IronDefense Pack v1.1.1 (Partner Supported)
#### Classifiers
##### IronDefense  
Added new Classifier

##### IronDefense - Incoming Mapper  
Added new incoming mapper

##### IronDefense - Classifier  
Added new Classifier

#### Integrations
##### IronDefense  
- Added ability to retrieve IronDefense alerts and events.
- Improving descriptions.

#### Layouts
- **layout-details-IronDefense_IronDome_Notification.json**
- **layout-details-IronDefense_Event_Notification.json**
- **layout-details-IronDefense_Alert_Notification.json**

---

### Malware Pack v1.2.3
#### Layouts

> - **layout-edit-Malware.json**

Reviewer: What changed?

- **layout-details-Malware.json**

---

### Malwarebytes Pack v1.0.2 (Partner Supported)
#### Integrations
##### Malwarebytes  
Added Code for Usage Analytics.

---

### McAfee ESM v10 and v11 Pack v1.0.3
#### Integrations
##### Deprecated: McAfee ESM v10 and v11  
Deprecated - We recommend using McAfee ESM v2 instead.

---

### Microsoft Graph User Pack v1.2.0
#### Integrations
##### Microsoft Graph User  
Fixed an issue where the ***msgraph-user-list*** command paging argument did not work as expected.

---

### Palo Alto Networks Cortex XDR - Investigation and Response Pack v2.3.3
#### Integrations
##### Palo Alto Networks Cortex XDR - Investigation and Response  
Added default classifier and mapper.

---

### PassiveTotal Pack v1.0.1
#### Integrations
##### PassiveTotal  
Added a limit to the number of related domains showed in the war room for enrichment commands.

---

### Phishing Pack v1.10.2
#### Layouts

> - **layout-quickView-Phishing.json**

Reviewer: What changed?

Author: Need to investigate why it happens on layouts only.

> # Cortex XSOAR Content Release Notes for version 20.8.2 (98126)

Reviewer 2: update version

##### Published on 1 September 2020

> ### New: EmailCommunication Pack v1.0.0

Reviewer 2: ```suggestion
### New: Email Communication Pack v1.0.0
```

Reviewer: We should also add a pack description for this one as it is a "use case" pack. We can copy the description from the pack:

> Do you have to send multiple emails to end users? This content pack helps you streamline the process and automate updates, notifications and more.

#### Integrations
##### Genians  
Use the Genians integration to block IP using assign tag and unassign tag.

---

### New: McAfee ESM Pack v1.0.0
#### Integrations
##### McAfee ESM v2  
Run queries and receive alarms from Intel Security ESM. Support version 10 and above.

---

### New: Microsoft Advanced Threat Analytics Pack v1.0.0
#### Classifiers
#####  Microsoft Advanced Threat Analytics - Classification  
Classifies Microsoft Advanced Threat Analytics suspicious activities
##### Microsoft Advanced Threat Analytics  

#####  Microsoft Advanced Threat Analytics - Incoming Mapper  
Maps Microsoft Advanced Threat Analytics suspicious activity fields.
#### IncidentFields
- **Suspicious Activity End Time**
- **Suspicious Activity ID**
- **Suspicious Activity Severity**
- **Suspicious Activity Start Time**
- **Suspicious Activity Status**
#### IncidentTypes
- **Microsoft ATA Suspicious Activity**
#### Integrations
##### Microsoft Advanced Threat Analytics  
Use Microsoft Advanced Threat Analytics integration to manage suspicious activities, monitoring alerts and entities.

---

## New: NTT Cyber Threat Sensor Pack v1.0.0 (Partner Supported)

> #### Classifiers!-->

Reviewer 2: ```suggestion
#### Classifiers
```

##### NTT Cyber Threat Sensor - Classifier  
Classifies NTT Cyber Threat Sensor incidents
##### NTT Cyber Threat Sensor  

##### NTT Cyber Threat Sensor - Incoming Mapper  
Maps incoming NTT Cyber Threat Sensor fields.
#### IncidentFields
- **FAERE Description**
- **Graph Plot**
#### IncidentTypes
- **TD Incident**
#### Integrations
##### NTT Cyber Threat Sensor  
Retrieve alerts and recommendations from NTT CTS
#### Playbooks
##### Handle TD events  
Playbook to enrich TD events

---

> ### New: PiHole Pack v1.0.0

Reviewer 2: this is community supported
@glicht do we want to add a comment about that?

Reviewer: Yes. We can add "Community Supported".

#### Integrations
##### PiHole  
Pi-hole is a network-level advertisement and Internet tracker blocking application which acts as a DNS sinkhole and optionally a DHCP server, intended for use on a private network.

---

### New: QueryAI Pack v1.0.0 (Partner Supported)
#### Integrations
##### Query.AI  
Query.AI is a decentralized data access and analysis technology that simplifies security investigations across disparate platforms without data duplication.

---

### Access Investigation Pack v1.2.2
#### Layouts
- **layout-edit-Access.json**
- **layout-details-Access.json**

---

### Base Pack v1.1.17
#### Scripts
##### WordTokenizerNLP  
Updated the script docker image to the latest version.

##### CommonServerPython  

> Added the following code objects, which simplifies creating widgets.

Reviewer 2: ```suggestion
- Added the following code objects, which simplifies creating widgets.
```

  - TextWidget
  - TrendWidget
  - NumberWidget
  - BarColumnPieWidget
  - LineWidget
  - TableOrListWidget
- Fixed an issue with mirroring mapper scheme.

> Fixed an issue in the **return_outputs** function where the content type was incorrect.

Reviewer 2: ```suggestion
- Fixed an issue in the **return_outputs** function where the content type was incorrect.
```

##### SaneDocReports  
Fixed an issue regarding the table readable headers.

---

### CheckPhish Pack v1.0.1
#### Integrations
##### CheckPhish  
- Updated the default API URL.
- Adjusted the error handling to use the new API format.

---

### Cisco AMP Pack v1.0.1
#### Integrations
##### Cisco AMP  
Fixed an issue where the `amp_move_computer` command would fail.

---

### Cisco Umbrella Investigate Pack v1.0.1
#### Integrations
##### Cisco Umbrella Investigate  
Fixed an issue where the ***umbrella-ip-dns-history*** command failed when no ip results were found.

---

### Code42 Pack v2.0.3 (Partner Supported)
#### Integrations
##### Code42  
Fix bug where File Category would not map correctly when creating incidents from Code42 alerts.

---

### Cofense Triage Pack v1.1.4 (Partner Supported)
#### Integrations
##### Cofense Triage v2  
- Fixed an issue in *cofense-get-attachment* command.

> - Internal code improvements.

Reviewer 2: lets remove that line

---

### Common Playbooks Pack v1.7.1
#### Playbooks
##### Extract Indicators From File - Generic v2  
Added support for UTF-8 Unicode text files.

---

### Common Scripts Pack v1.2.34
#### Scripts
##### WhereFieldEquals  
Fixed an issue where `WhereFieldEquals` would return a string instead of a list.

##### FeedRelatedIndicatorsWidget  
Fixed an issue where the indicator link value was incorrect.

---

### Common Types Pack v1.8.8
#### Layouts
- **File Indicator**
- **layout-edit-Vulnerability.json**
- **layout-details-Vulnerability.json**

---

### CrowdStrike Falcon Streaming Pack v1.0.7
#### Integrations
##### CrowdStrike Falcon Streaming v2  
- Improved error handling of unsupported media types.
- Improved handling when the stream response client is not completed.
- Maintenance and stability enhancements.

---

### CyberArk Pack v1.0.2
#### Integrations
##### CyberArk PAS  
- Added documentation for the integration.
- Updated the the **cyberark-pas-credentials-verify** command to use the new API.

> Added the ***cyberark-pas-account-get-details*** command.

Reviewer 2: ```suggestion
- Added the ***cyberark-pas-account-get-details*** command.
```

##### Deprecated: Endace Search Archive Download PCAP  
This playbook is deprecated. Use Endace Search Archive Download PCAP v2 instead.

##### Endace Search Archive Download PCAP v2  
Added support for directionless ip and port search, user friendly timeframe values, updated input and output variables to the playbook and their definitions.

---

### Expanse Pack v1.1.1 (Partner Supported)
#### Integrations
##### Expanse  
Updated the version number in the user-agent header

---

### GDPR Pack v1.0.5
#### Layouts
- **layout-edit-GDPR_Data_Breach.json**
- **layout-details-GDPR_Data_Breach.json**

---

### GitHub Pack v1.1.1
#### Integrations
##### GitHub  
Improved implementation of parameters errors handling.

---

### Gmail Pack v1.0.5
#### Integrations
##### Gmail  
Breaking Change: Removed the deprecated *event* argument from the ***gmail-list-users*** command.

---

### IBM QRadar Pack v1.0.8
#### Integrations
##### IBM QRadar  
Added a parameter that enables you to specify the number of addresses to enrich per API call.

---

### Infocyte Pack v1.0.2 (Partner Supported)
#### Integrations
##### Infocyte  

> Maintenance and stability enhancements.

Reviewer 2: lets remove that pack from the rn

---

### Palo Alto Networks Cortex XDR - Investigation and Response Pack v2.3.3
#### Integrations
##### Palo Alto Networks Cortex XDR - Investigation and Response  
Added default classifier and mapper.

---

### PassiveTotal Pack v1.0.1
#### Integrations
##### PassiveTotal  
Added a limit to the number of related domains showed in the war room for enrichment commands.

---

### Phishing Pack v1.10.2
#### Layouts
- **layout-quickView-Phishing.json**
- **layout-mobile-Phishing.json**
- **Phishing Incident**
- **layout-edit-Phishing.json**

---

### Proofpoint TAP Pack v1.1.0
#### Integrations
##### Proofpoint TAP v2  
Fixed a bug in the ***proofpoint-get-forensics*** command which caused getting a threat by *campaignId* to fail.

---

### ServiceNow Pack v1.2.1
#### Classifiers
##### New: ServiceNow Classifier  
Classifies ServiceNow tickets.

##### New: ServiceNow - Outgoing Mapper  
Maps outgoing ServiceNow incident Fields.

##### New: ServiceNow - Incoming Mapper  
Maps incoming ServiceNow incidents fields.

#### Integrations
##### ServiceNow v2  
- Added the ticket mirroring feature.
- Removed unnecessary import.

> #### Layouts

Reviewer 2: need to add the layout here
@yaakovi any idea why it was generated empty ?

---

### SplunkPy Pack v1.1.3
#### Integrations
##### SplunkPy  
- Fixed an issue where the ***splunk-parse-raw*** command failed when the *raw* argument received input in json format.
- Fixed an issue where ***fetch-incidents*** would miss some incidents.

---

### Symantec Endpoint Protection Pack v1.0.2
#### Integrations
##### Symantec Endpoint Protection v2  
Fixed an issue in which the *domain* parameter was not parsed correctly.

---

### Tanium Pack v1.0.2
#### Integrations
##### Tanium v2  

> - Added completion-percantage argument to the commands **tn-get-saved-question-result** and **tn-get-question-result**

Reviewer 2: ```suggestion
Added completion-percantage argument to the commands **tn-get-saved-question-result** and **tn-get-question-result**
```
